### DISCUSSION TITLE
Next-intl Localization error

### DISCUSSION THREAD

**Discussion by Voetbaltrips** [amannn/next-intl#1123] [Q&A]
### Description

I'm trying to use App Router setup without i18n routing approach to do localization in my app but everything fine till I reach this point 
![image](https://github.com/amannn/next-intl/assets/165676987/5f86e433-743a-4e0b-a4cd-739a09f68027)

then starting to get this error in the console

![image](https://github.com/amannn/next-intl/assets/165676987/bd4568bf-d97e-423d-aa8d-e72d2d1f11a4)

hree is me i18n.js file

![image](https://github.com/amannn/next-intl/assets/165676987/ebc93dc3-ab26-40e9-9935-382a1b293460)

so what is wrong with my code.
Thank you.

### Verifications

- [X] I've verified that the problem I'm experiencing isn't covered in [the docs](https://next-intl-docs.vercel.app/).
- [X] I've searched for similar, existing issues on GitHub and [Stack Overflow](https://stackoverflow.com/search?q=next-intl).
- [X] I've compared my app to [a working example](https://next-intl-docs.vercel.app/examples) to look for differences.

### Mandatory reproduction URL

https://visitfootball.vercel.app/

### Reproduction description

Steps to reproduce:
1. Open reproduction
2. Click on …
3. See error: … 
This is cannot be reproduction as its not starting at lal

### Expected behaviour

To work

**Comment by amannn** [comment]
Can you ensure you're on the latest version of `next-intl`? The stack trace looks a bit fishy to me. Maybe reinstall your `node_modules` when in doubt.

If that doesn't help, can you add logging to see the result for the `locale` you've negotiated in `i18n.js`?

I'll move this to a discussion since it's a usage question and not a bug report.
